Classes/UVSuggestionSearchResultsController.m

```diff
@@ -0,0 +1,159 @@
+//
+//  UVSuggestionSearchResultsController.m
+//  UserVoice
+//
+//  Created by Donny Davis on 9/13/16.
+//  Copyright © 2016 UserVoice Inc. All rights reserved.
+//
+
+#import "UVSuggestionSearchResultsController.h"
+#import "UVUtils.h"
+#import "UVClientConfig.h"
+#import "UVSession.h"
+#import "UVSuggestion.h"
+#import "UVSuggestionDetailsViewController.h"
+
+#define TITLE 20
+#define SUBSCRIBER_COUNT 21
+#define STATUS 22
+#define STATUS_COLOR 23
+
+@interface UVSuggestionSearchResultsController ()
+
+@end
+
+@implementation UVSuggestionSearchResultsController
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        [self setupPlainTableView];
+        return self;
+    }
+}
+
+- (void)dealloc {
+    _searchResults = nil;
+}
+
+- (void)dismiss {
+    [super dismiss];
+}
+
+#pragma mark ===== table cells =====
+
+- (void)initCellForResult:(UITableViewCell *)cell indexPath:(NSIndexPath *)indexPath {
+    [self initCellForSuggestion:cell indexPath:indexPath];
+}
+
+- (void)customizeCellForResult:(UITableViewCell *)cell indexPath:(NSIndexPath *)indexPath {
+    [self customizeCellForSuggestion:[_searchResults objectAtIndex:indexPath.row] cell:cell];
+}
+
+- (void)initCellForSuggestion:(UITableViewCell *)cell indexPath:(NSIndexPath *)indexPath {
```

Comment:
It appears that this method and the one below it are copied verbatim from UVSuggestionListViewController. Could you extract them into a helper so that the function bodies are not duplicated?